WalletWasabi.Tests/HwiTests/NoDeviceConnectedTests/HwiProcessBridgeMock.cs

```diff
@@ -0,0 +1,310 @@
+			}
+			else if (Model == HardwareWalletModels.LedgerNanoS)
+			{
+				type = "ledger";
+				rawPath = "\\\\\\\\?\\\\hid#vid_2c97&pid_0001&mi_00#7&e45ae20&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}";
+			}
+			else
+			{
+				throw new NotImplementedException("Mock missing");
+			}
+
+			string path = HwiParser.NormalizeRawDevicePath(rawPath);
+			string devicePathAndTypeArgumentString = $"--device-path \"{path}\" --device-type \"{type}\"";
+
+			const string successTrueResponse = "{\"success\": true}\r\n";
+
+			if (CompareArguments(arguments, "enumerate"))
+			{
+				if (Model == HardwareWalletModels.TrezorT)
+				{
+					var response = $"[{{\"type\": \"{type}\", \"path\": \"{rawPath}\", \"needs_pin_sent\": false, \"needs_passphrase_sent\": false, \"error\": \"Not initialized\"}}]";
+					var code = 0;
+					return Task.FromResult((response, code));
+				}
+				else if (Model == HardwareWalletModels.TrezorOne)
+				{
+					var response = $"[{{\"type\": \"{type}\", \"path\": \"{rawPath}\", \"needs_pin_sent\": true, \"needs_passphrase_sent\": false, \"error\": \"Could not open client or get fingerprint information: Trezor is locked. Unlock by using 'promptpin' and then 'sendpin'.\", \"code\": -12}}]\r\n";
+					var code = 0;
+					return Task.FromResult((response, code));
+				}
+				else if (Model == HardwareWalletModels.ColdcardMk1)
+				{
+					var response = $"[{{\"type\": \"{type}\", \"path\": \"{rawPath}\", \"needs_passphrase\": false, \"fingerprint\": \"a3d0d797\"}}]\r\n";
+					var code = 0;
+					return Task.FromResult((response, code));
+				}
+				else if (Model == HardwareWalletModels.LedgerNanoS)
+				{
+					var response = $"[{{\"type\": \"{type}\", \"path\": \"{rawPath}\", \"fingerprint\": \"4054d6f6\", \"needs_pin_sent\": false, \"needs_passphrase_sent\": false}}]\r\n";
+					var code = 0;
+					return Task.FromResult((response, code));
+				}
+			}
+			else if (CompareArguments(arguments, $"{devicePathAndTypeArgumentString} wipe"))
+			{
+				if (Model == HardwareWalletModels.TrezorT || Model == HardwareWalletModels.TrezorOne)
+				{
+					var response = successTrueResponse;
+					var code = 0;
+					return Task.FromResult((response, code));
+				}
+				else if (Model == HardwareWalletModels.ColdcardMk1)
+				{
+					var response = "{\"error\": \"The Coldcard does not support wiping via software\", \"code\": -9}\r\n";
+					var code = 0;
+					return Task.FromResult((response, code));
+				}
+				else if (Model == HardwareWalletModels.LedgerNanoS)
+				{
+					var response = "{\"error\": \"The Ledger Nano S does not support wiping via software\", \"code\": -9}\r\n";
+					var code = 0;
+					return Task.FromResult((response, code));
+				}
+			}
+			else if (CompareArguments(arguments, $"{devicePathAndTypeArgumentString} setup"))
+			{
+				if (Model == HardwareWalletModels.TrezorT || Model == HardwareWalletModels.TrezorOne)
+				{
+					var response = "{\"error\": \"setup requires interactive mode\", \"code\": -9}";
```

Comment:
Setup with capital S.

Reply:
This is a mock. We are reproducing the exact same response that a hardware wallet and HWI would give us to be able to test our code without a hardware wallet plugged in. So we cannot change these here.